### DISCUSSION TITLE
Can I use multiple of the same key?

### DISCUSSION THREAD

**Discussion by jsaunders92** [47ng/nuqs#514] [Q&A] [answered]
I know I'm probably being an idiot here and missing something but if I wanted to use:

?filter=xyz&filter=abc

Is that actually possible?

**Comment by franky47** [comment, marked as answer]
Not currently, but there is a [discussion](https://github.com/47ng/nuqs/discussions/484) about bringing support, feedback welcome.

**Reply by jsaunders92** [reply]
Thank you for such a very quick reply! I will certainly have a look at the discussion.
